### ISSUE TITLE
tcmalloc: large alloc warnings

### ISSUE THREAD

**Issue by Gladiator07** [open-mmlab/mmdetection#6672]
I am training an instance segmentation model on colab pro (25GB RAM, Tesla P100 GPU 16GB) and I am getting the following warning. Although, the training continues and there are no problems at all. Should I take this warning seriously or just ignore it for now ?
![image](https://user-images.githubusercontent.com/52319020/144482373-73e29a5d-79a5-4254-a122-6cca7728d327.png)

**Comment by BIGWangYuDong** [comment]
Hi, Seems that you can ignore this.

**Comment by Gladiator07** [comment]
Thanks. I found some information about this warning. Basically, it is an environment variable, which is by default set to 1GB. If there is allocation that exceeds it, this message is displayed. This is expected in memory heavy use cases. I have attached the resources to read more about the warning, incase if anyone faces the same issue. Closing this issue.

- [Stackoverflow answer](https://stackoverflow.com/a/52399665/11775511)
- [PyTorch forums](https://discuss.pytorch.org/t/tcmalloc-output-when-training-with-large-dataseta/103376/2?u=atharva_ingle)
